src/DynamoCore/Updates/UpdateManager.cs

```diff
@@ -78,6 +78,10 @@ public interface IUpdateManager
         /// </summary>
         BinaryVersion AvailableVersion { get; }
 
+        Version HostVersion { get; set; }
```

Comment:
We need to define a new interface IUpdateManager2 or IUpdateManagerHost or IDynamoHostApplication otherwise, it would break binary compatibility. You can try it yourself, run Dynamo Studio 1.1 with your DynamoCore.